lr-cli/cortx_setup/commands/__init__.py

```diff
@@ -72,5 +73,7 @@
     'ResourceDiscover',
     'SecurityConfig',
     'SetSignature',
-    'StorageEnclosureConfig'
-]
+    'StorageEnclosureConfig',
+    'SaltCleanup',
+    'PillarSync'
```

Comment:
PillarSync is repeated twice. Remove one to avoid confusion.

Reply:
I think this was missed earlier so added here